rpm/pacemaker.spec.in

```diff
@@ -274,6 +274,9 @@ Requires:      resource-agents
 Requires:      %{pkgname_pcmk_libs}%{?_isa} = %{version}-%{release}
 Requires:      %{name}-cluster-libs%{?_isa} = %{version}-%{release}
 Requires:      %{name}-cli = %{version}-%{release}
+%if %{with stonithd}
```

Comment:
We also need to guard packaging fence_legacy and its man page

Reply:
Fixed.